test/test_genome.py

```diff
@@ -404,6 +404,30 @@ def test_mutate_output_region(rng_seed):
         ID_NON_CODING_GENE,
     ]
 
-    assert genome._mutate_output_region(list(dna), 9, rng) is False
-    assert genome._mutate_output_region(list(dna), 10, rng) is True
-    assert genome._mutate_output_region(list(dna), 11, rng) is False
+    assert genome._mutate_output_region(list(dna), 9, rng) is True
+    assert genome._mutate_output_region(list(dna), 10, rng) is False
+    assert genome._mutate_output_region(list(dna), 11, rng) is True
+
+
+def test_correct_number_of_mutations(mutation_rate, rng_seed):
```

Comment:
I think it would be better to test this with e.g. three mutation rates, instead of just one, using `parametrize` fixture of `pytest`.

Reply:
In the newest commit I added: `@pytest.mark.parametrize("mutation_rate", [0.02, 0.05, 0.2,])` just before this test. Is that the right way to do it?

Reply:
Looks good to me, although I am not sure whether this might interfer with the fixture from `conftest.py` that has the same name.